pkgs/servers/firebird/4.nix

```diff
@@ -0,0 +1,59 @@
+{ autoreconfHook
+, fetchFromGitHub
+, icu
+, lib
+, libedit
+, libtomcrypt
+, libtommath
+, stdenv
+, unzip
+, zlib
+
+, superServer ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firebird";
+  version = "4.0.2";
+
+  src = fetchFromGitHub {
+    owner = "FirebirdSQL";
+    repo = "firebird";
+    rev = "v${version}";
+    sha256 = "sha256-hddW/cozboGw693q4k5f4+x9ccQFWFytXPUaBVkFnL4=";
+  };
+
+  configureFlags = [
+    "--with-system-editline"
+  ] ++ (lib.optional superServer "--enable-superserver");
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ icu ];
```

Comment:
Where is that required? It would be nicer if it would be in the prePhase in bash code.

Reply:
If I comment errors as:
Firebird 3: https://termbin.com/lv2o

Reply:
I was able to build using:
```
  preBuild = ''
    export LD_LIBRARY_PATH=${lib.makeLibraryPath [ icu ]}
  '';
```

Reply:
That shouldn't be required but I don't really have time to dig into this.

Reply:
> /build/source/gen/Release/firebird/bin/isql -q -i /build/source/src/dbs/metadata.sql
can't format message 17:0 -- message file /build/source/gen/Release/firebird/firebird.msg not found
Could not find acceptable ICU library

Somewhere in the build script it calls this.

Reply:
We are better not digging into this.

Reply:
Regression from preBuild change affected runtime. Need to revert.